Cargo.toml

```diff
@@ -12,7 +12,7 @@ categories = ["network-programming", "asynchronous"]
 exclude = [".gitignore", ".github/*"]
 
 [dependencies]
-enr = { version = "0.5.1", features = ["k256", "ed25519"] }
```

Comment:
needs to be updated pending https://github.com/rust-ethereum/enr/pull/33#pullrequestreview-959941878